dash_core_components_base/__init__.py

```diff
@@ -116,19 +116,19 @@
         'async': 'eager'
     },
     {
-        'relative_package_path': 'async~plotlyjs.js',
+        'relative_package_path': 'async-plotlyjs.js',
         'external_url': (
             'https://unpkg.com/dash-core-components@{}'
-            '/dash_core_components/async~graph~plotlyjs.js'
+            '/dash_core_components/async-plotlyjs.js'
```

Comment:
Pretty bad error here and below with `serve_locally=False`

Reply:
Yikes, nice find! Is there anything we can do to test this? I'm thinking like a standard test we can put in `dash.testing` and then run in all our repos with appropriate parameters, looping over `_js_dist` and ensuring `external_url` matches a certain pattern vs `relative_package_path`?

Reply:
a followup issue would be fine for now.

Reply:
Follow up in https://github.com/plotly/dash-core/issues/171 - providing at least two approaches: pattern matching as suggested and something more robust / matching e2e usage but requiring more effort.